hub/src/server/config.ts

```diff
@@ -141,14 +141,15 @@ export function getConfig() {
 
   const formats = [
     config.argsTransport.colorize ? winston.format.colorize() : null,
-    config.argsTransport.json ? winston.format.json() : null,
-    config.argsTransport.timestamp ? winston.format.timestamp() : null
+    config.argsTransport.timestamp ?
```

Comment:
Winston lib had a ton of changes, nice fix here. This is just getting the output to match the previous version output right? 
Also, I have this code block copy-pasted in all three sub-projects. Not happy with it, but didn't want to introduce cross-project dependencies in the PR (aside from the unit tests where it seems to work well). 
Should we update this in the other projects, or make a fourth sub-project called `common` that we can put this kind of stuff in, then cross reference? Typescript seems okay with path traversals outside of the `src` dir, so we wouldn't need to hacky package.json `"common-utils": "file:../common-utils"` NPM import.

Reply:
I'll just update the other projects for now. We should probably pull things like this into a common sub-project at some point though.